docs/stories/03-inputs/Combobox.stories.tsx

```diff
@@ -189,6 +190,63 @@ export const Creatable: Story = {
   },
 };
 
+export const CreatableVisible: Story = {
+  render: () => {
+    const [value, setValue] = React.useState<string | undefined>('');
+
+    const onCreateOption = () => {};
+
+    return (
+      <Combobox
+        aria-describedby="creatable combobox"
+        placeholder="My favourite fruit is..."
+        value={value}
+        onChange={setValue}
+        onCreateOption={onCreateOption}
+        creatable="visible"
+        creatableStartIcon={<Plus fill="neutral500" />}
+        createMessage={() => 'Create a relation'}
+      >
+        {options.map(({ name, value }) => (
+          <ComboboxOption key={value} value={value}>
+            <Flex gap={2} justifyContent="space-between">
+              <Flex gap={2}>
+                <LinkIcon fill="neutral500" />
+                <Typography ellipsis>{name}</Typography>
+              </Flex>
+            </Flex>
+          </ComboboxOption>
+        ))}
+      </Combobox>
+    );
+  },
+  name: 'Creatable Visible',
+  parameters: {
+    docs: {
+      source: {
+        code: outdent`
+      <Combobox
+        aria-describedby="creatable combobox"
+        placeholder="My favourite fruit is..."
+        value={value}
+        onChange={setValue}
+        onCreateOption={onCreateOption}
+        creatable="visible"
+        creatableStartIcon={<Plus fill="neutral500" />}
+        createMessage={() => 'Create a relation'}
```

Comment:
Please don't mention any CMS features in the DS.

Reply:
especially when the combobox is about Fruits actually (placeholder, options, etc) 😅